### ISSUE TITLE
 computation of full Hilbert spectrum 

### ISSUE THREAD

**Issue by ggrrll** [jaidevd/pyhht#55]
Hi, 

if I understood correctly at the moment there is only the possibility to get the insta. freq. and not the [whole Hilbert spectrum](https://en.wikipedia.org/wiki/Hilbert_spectrum), right? 

are there progress/motivations in that direction?

**Comment by jaidevd** [comment]
Yes, that is what I'm waiting to finish before I can make a stable release. Any help is appreciated.

**Comment by ggrrll** [comment]
more specifically where? [in this notebook](https://github.com/jaidevd/pyhht/blob/jd-add-hhs/hhs.ipynb)?

**Comment by jaidevd** [comment]
Yes. Except it needs to be optimized, tested for correctness and documented.
